stylesheets/style.css

```diff
@@ -1,7 +1,258 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+  line-height: 1;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
 /*
 background-grey: #F4F3F4;
 button-purple: #192592
 main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+
+
+@media screen and (max-width: 768px){
+
+  
+  body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    clear: both;
+    width: 100%;
+  }
+  
+  nav {
+    order: -1;
+    position: fixed;
+    top: 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: center;
+    background: white;
+    border-bottom: 1px solid rgba(44, 48, 63, 0.452);
+    
+  }
+
+  nav img {
+    width: 30%;
+    height: 30%;
+    
+  }
+
+  nav li {
+    display: none;
+  }
+
+header {
+  display: flex;
+  flex-direction: column;
+  margin-top: 100px; /*how can i put it to the next position without using margin? with flex column?*/
+  text-align: left;
+  align-items: center;
+  width: 100%;
+}
+
+
+h1{
+
+  color: #2C303F;
+  font-size: 35px;
+  font-weight: bold;
+  margin: 0 15px 15px 15px;
```

Comment:
Try to find the way no to repeat code in CSS neither. So in this case `margin: 0 15px 15px 15px;` is the same in the `h1` and `p`, maybe you could find a way to put it in both with one rule?